static/js/components/LearningResourceCard.js

```diff
@@ -174,8 +177,15 @@ export function LearningResourceDisplay(props: Props) {
             {readableLearningResources[object.object_type]}
           </div>
           <div className="audience-certificates">
-            {icons.length > 0
-              ? icons.map((icon, i) => <img src={icon} key={i} />)
+            {iconKeys.length > 0
+              ? iconKeys.map((key, i) => (
+                <span className="facet-icon" key={i}>
+                  <span className="icon-tooltip-text">
+                    {tooltipTextMap[key]}
+                  </span>
+                  <img src={iconMap[key]} key={i} />
+                </span>
```

Comment:
what do you think about refactoring this into a `LearningResourceBadge` component (or similar name, `LearningResourceIcon`, idk) which would just take a `iconKey` prop? would let us re-use is here and on the `SearchFacetItem`